doc/sphinx/proof-engine/tactics.rst

```diff
@@ -1676,68 +1615,100 @@ Controlling the proof flow
    .. exn:: @ident is used in conclusion.
       :undocumented:
 
-.. tacn:: generalize @term
-   :name: generalize
-
-   This tactic applies to any goal. It generalizes the conclusion with
-   respect to some term.
+   .. tacn:: specialize_eqs @ident
+      :undocumented:
 
-.. example::
+.. tacn:: generalize @one_term {? {+ @one_term } }
+          generalize @one_term {? at @occs_nums } {? @as_name } {* , @pattern_occs {? @as_name } }
```

Comment:
Why not the following? (This is equivalent.)
```suggestion
          generalize {+, @pattern_occs {? @as_name } }
```

Reply:
Note I moved the `occs_nums` description to `pattern_occs`

Reply:
> Sure. And then only need a single `:tacn:` here.

Unless I'm mistaken, I don't think so. One form uses comma separators and the other not.

Reply:
Good point, weird syntax, my bad.

Reply:
I also find it weird, but I bet there is an explanation. Likely the first form was initially available, but adding new optional clauses made it necessary to have an explicit separator.

Reply:
A reasonable guess, but it's still unambiguous without any commas.  Though maybe more readable for humans with commas for grouping.